pkg/controller/clusterstorage/clusterstorage_controller.go

```diff
@@ -192,7 +193,8 @@ func (r *ReconcileClusterStorage) syncStatus(clusterOperator *configv1.ClusterOp
 	}
 	v1helpers.SetStatusCondition(&clusterOperator.Status.Conditions, notProgressing)
 
-	if err != nil {
+	// if error is anything other than unsupported platform, we are failing
+	if err != nil && err != unsupportedPlatformError {
```

Comment:
+1: it could be useful to have a log entry if `err == unsupportedPlatformError`.

Reply:
fixeds